src/client/datascience/interactive-ipynb/nativeEditorProviderOld.ts

```diff
@@ -378,7 +378,7 @@ export class NativeEditorProviderOld extends NativeEditorProvider {
             (editorUri) =>
                 editorUri.document &&
                 editorUri.document.uri.scheme === 'git' &&
-                this.fs.arePathsSame(editorUri.document.uri, editor.document.uri)
+                editorUri.document.uri.fsPath === editor.document.uri.fsPath
```

Comment:
```suggestion
                editorUri.document.uri.fsPath === editor.document.uri.fsPath
```

Adding a comment would be useful & probably adding a test to ensure this works with different resource schemes (I'm assuming that's the case here).

Reply:
I.e. we need to ensure we do not regress again, comment + test would be useful.

Reply:
Oh, sorry, merged this before this comment showed up! I'll put up a new PR @DonJayamanne